### DISCUSSION TITLE
高分辨率设置的支持

### DISCUSSION THREAD

**Discussion by chloerei** [geeknote-net/feedback#33] [General]
目前对高分辨率设置的支持不太好，字体太小内容太窄。但我不知道处理这个问题的最佳方法是什么，参考同类网站在高分辨率下似乎也没有做特别处理，希望大家提供建议，如果有的话提供现成网站或应用参考。

1920*1080:
<img width="1920" alt="截屏2023-03-08 14 31 39" src="https://user-images.githubusercontent.com/49931/223640158-878c1ef7-b772-40ef-b6ce-48e23b971d13.png">

2560*1440
<img width="2560" alt="截屏2023-03-08 14 51 14" src="https://user-images.githubusercontent.com/49931/223640201-1b2828c9-a82d-42fc-965d-d9e0d4e9d1a8.png">

**Comment by linvaux** [comment]
语雀之前也是只能窄屏编辑，后面加了一个“宽屏编辑”的按钮，可以放大编辑区域
